src/include/planner/operator/persistent/logical_copy_to.h

```diff
@@ -25,25 +20,16 @@ class LogicalCopyTo : public LogicalOperator {
     void computeFactorizedSchema() override;
     void computeFlatSchema() override;
 
-    std::string getFilePath() const { return filePath; }
-    common::FileType getFileType() const { return fileType; }
-    std::vector<std::string> getColumnNames() const { return columnNames; }
-    const std::vector<common::LogicalType>& getColumnTypesRef() const { return columnTypes; }
-    const common::CSVOption* getCopyOption() const { return &copyToOption; }
-    bool getCanParallel() const { return canParallel; }
+    std::unique_ptr<function::CopyFuncBindData> getBindData() const { return bindData->copy(); }
+    function::CopyFunction getCopyFunc() const { return copyFunc; };
 
     std::unique_ptr<LogicalOperator> copy() override {
-        return make_unique<LogicalCopyTo>(filePath, fileType, columnNames, columnTypes,
-            copyToOption.copy(), children[0]->copy(), canParallel);
+        return make_unique<LogicalCopyTo>(bindData->copy(), copyFunc, children[0]->copy());
     }
 
 private:
-    std::string filePath;
-    common::FileType fileType;
-    std::vector<std::string> columnNames;
-    std::vector<common::LogicalType> columnTypes;
-    common::CSVOption copyToOption;
-    bool canParallel;
+    std::unique_ptr<function::CopyFuncBindData> bindData;
```

Comment:
You can use object here I guess.

Reply:
bindData is a virtual class, we have to hold a unique ptr to it